studies/migrations/0074_add_confirmed_to_response.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.0.14 on 2021-07-12 19:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("studies", "0073_add_study_type_to_response")]
+
+    operations = [
+        migrations.AddField(
+            model_name="response",
+            name="confirmed",
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
might want to squash with previous migration (at the point of actually merging in). Hooray for the first steps towards external studies today!